src/screens/OrganizationDashboard/OrganizationDashboard.test.tsx

```diff
@@ -101,4 +104,36 @@ describe('Organisation Dashboard Page', () => {
     const image = screen.getByTestId(/orgDashImgAbsent/i);
     expect(image).toBeInTheDocument();
   });
+
+  test('should check organization delete call', async () => {
+    const { container } = render(
+      <MockedProvider addTypename={false} mocks={MOCKS_WITHOUT_IMAGE}>
+        <BrowserRouter>
+          <Provider store={store}>
+            <I18nextProvider i18n={i18nForTest}>
+              <OrganizationDashboard />
+            </I18nextProvider>
+          </Provider>
+        </BrowserRouter>
+      </MockedProvider>
+    );
+
+    expect(container.textContent).not.toBe('Loading data...');
+    await wait();
+    fireEvent.click(screen.getByText('Delete This Organization'));
+    const orgInput = screen.getByTestId(/orgName/i);
+    userEvent.type(orgInput, 'bar');
+    expect(screen.getByTestId(/deleteOrganizationBtn/i)).toHaveProperty(
+      'disabled',
+      true
+    );
+
+    userEvent.clear(orgInput);
+    userEvent.type(orgInput, 'Dummy Organization');
```

Comment:
@palisadoes @kayceeDev userEvent.type is not triggering the onChange event in the test, but works fine locally.
I would appreciate any assistance with this.

Reply:
What have you tried?

Reply:
https://github.com/testing-library/user-event/issues/676
I have tried all solutions here @Rumone

Reply:
> Adding userEvent.tab() after the userEvent.type() also works. 

You tried this as well.

Reply:
yes i did that also